### DISCUSSION TITLE
Repository for ohmjs.org?

### DISCUSSION THREAD

**Discussion by WalterGR** [ohmjs/ohm#365] [Help] [answered]
Is there a repository for [https://ohmjs.org/](https://ohmjs.org/)?

The "Ohm Editor" link in the "🦸 Online editor and visualizer" section is a 404.  It needs a trailing slash.

**Comment by pdubroy** [comment, marked as answer]
Thanks @WalterGR! Just fixed it.

The repo is at https://github.com/ohmlang/ohmlang.github.io btw
